embulk-input-mysql/build.gradle

```diff
@@ -13,3 +13,38 @@ embulkPlugin {
     category = "input"
     type = "mysql"
 }
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                developers {
+                    developer {
+                        name = "Sadayuki Furuhashi"
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Hitoshi Tanaka"
+                        email = "thitoshi@cac.co.jp"
+                    }
+                    developer {
+                        name = "Muga Nishizawa"
+                        email = "muga.nishizawa@gmail.com"
+                    }
+                    developer {
+                        name = "Hiroyuki Sato"
+                        email = "hiroysato@gmail.com"
+                    }
+                    developer {
+                        name = "Dai MIKURUBE"
+                        email = "dmikurube@treasure-data.com"
+                    }
+                    developer {
+                        name = "Robert Nguyen"
+                        email = "ng.hung83@gmail.com"
+                    }
```

Comment:
@tvhung83 Hi!

We wonder if you're okay to nominate you in `pom.xml` of `embulk-input-mysql` on Maven Central, from your work in #153. I picked up your name and email from the Git commit log, which are already public.

Please let me know if you don't want, or if you want another name/email.

Reply:
@dmikurube sure, that'd be my honor! Thank you for reaching out.